python/hail/methods/qc.py

```diff
@@ -341,12 +341,12 @@ def concordance(left, right) -> Tuple[List[List[int]], Table, Table]:
     return global_conc, col_conc, row_conc
 
 
-@typecheck(dataset=MatrixTable,
+@typecheck(dataset=oneof(Table, MatrixTable),
            config=str,
            block_size=int,
            name=str,
            csq=bool)
-def vep(dataset, config, block_size=1000, name='vep', csq=False) -> MatrixTable:
+def vep(dataset: Union[Table, MatrixTable], config, block_size=1000, name='vep', csq=False):
```

Comment:
Do you want ` -> Union[Table, MatrixTable]`?  That's what I did for `filter_intervals`.

Reply:
Ah, since it was a Union, I thought it might be OK in that case since it could be OK.  Leave it, and I'll remove mine from `filter_intervals`.